src/main/java/com/uid2/attestation/gcp/OidcAttestationProvider.java

```diff
@@ -0,0 +1,32 @@
+package com.uid2.attestation.gcp;
+
+import com.uid2.enclave.AttestationException;
+import com.uid2.enclave.IAttestationProvider;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class OidcAttestationProvider implements IAttestationProvider {
+	// the local file contains OIDC token
+	private String tokenFilePath = "/run/container_launcher/attestation_verifier_claims_token";
```

Comment:
Shoud we put this tokenFilePath to ctor?

Reply:
👍 updated